packages/solutions/app-tools/src/analyze/getBundleEntry.ts

```diff
@@ -64,6 +64,7 @@ export const getBundleEntry = (
           ? {
               entryName: name,
               entry: ensureAbsolutePath(appDirectory, value),
+              entryDir: path.dirname(ensureAbsolutePath(appDirectory, value)),
```

Comment:
set this value to `absoluteEntryDir` could be enough

Reply:
otherwise `entryDir` could be empty in nested routes analyzed entries

Reply:
fixed this with make the getBundleEntry calculate the absoluteEntryDir too.
The problem in nested routes entries not have absoluteEntryDir is because in nested route:
![image](https://user-images.githubusercontent.com/13596193/218958481-95dac570-1f90-4ef7-b176-2f989808c490.png)